packages/testing/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "@redhat-cloud-services/frontend-components-testing",
+    "version": "1.0.0",
+    "description": "Testing utilities for RedHat Cloud Services project.",
+    "main": "index.js",
+    "module": "esm/index.js",
+    "types": "index.d.ts",
+    "sideEffects": false,
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "build": "npm run build:js && npm run build:esm && npm run build:packages",
+        "build:packages": "node ../../scripts/build-packages.js --forceTypes",
+        "build:esm": "npx tsc --module es2015 --target es5",
+        "build:js": "npx tsc -p tsconfig.cjs.json"
```

Comment:
The `npx` should not be used and is being fixed in another PR. Can you use TSC directly, please? For reference: 2c76dc06185f93bec73660e15338433b95eac6707317564c0f778f5b9abe446cR17